src/WAV.jl

```diff
@@ -656,7 +657,7 @@ function get_default_precision(samples, compression)
 end
 
 function wavwrite(samples::AbstractArray, io::IO; Fs=8000, nbits=0, compression=0,
-                  chunks::Dict{Symbol, Array{UInt8,1}}=Dict{Symbol, Array{UInt8,1}}())
+                  chunks::Vector{Tuple{Symbol, Array{UInt8,1}}}=Tuple{Symbol, Array{UInt8,1}}[])
```

Comment:
Is it possible to write adapter code? That would make this a non-breaking change. It appears that you could flatten the incoming dictionary and call your new interface.

Reply:
I agree it would be nice for it to be non-breaking. What should happen in the `wavread()` case? If it returns a dictionary then you can't recover multiple chunks with the same identifier. So should it stay as returning a list of tuples?

Reply:
This does look tricky. I think that there are two plausible options:
1. use what you have here and release this as version 1.0.0
2. add a new function name that uses your new data type. The existing wavwrite and wavread functions could call the new function.

What do you think?

Regarding INFO and CUE, I would gladly accept those additions.

Reply:
I'd prefer option 1, but we could do both? Release 0.10.0 with a new function as well as the old ones, and release 1.0.0 with these changes. 

And awesome :) I'll work on bringing INFO and CUE in.